srcpkgs/dracut/template

```diff
@@ -14,17 +14,25 @@ license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="http://www.kernel.org/pub/linux/utils/boot/dracut/dracut.html"
 distfiles="https://github.com/dracutdevs/dracut/archive/${version}.tar.gz"
 checksum=f9dbf18597e5929221365964293212c8c9ffb7d84529c5a338c834ecab06e333
+subpackages="dracut-network"
 
 make_dirs="
-	/etc/dracut.conf.d 0755 root root
-	/usr/lib/dracut/dracut.conf.d 0755 root root"
+ /etc/dracut.conf.d 0755 root root
+ /usr/lib/dracut/dracut.conf.d 0755 root root"
 
 case "$XBPS_TARGET_MACHINE" in
 *-musl) # Add musl-fts implementation
 	makedepends+=" musl-fts-devel"
 	;;
 esac
 
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*|aarch64*) # EFI bundle config and hooks
+		_efi_bundle=1
+		subpackages+=" dracut-uefi"
```

Comment:
`uefi-bundle` is not supported in aarch64. (`/usr/bin/dracut:1140`)

Reply:
Oh, good find. Wonder if they forgot to add aarch64 or if it doesn't work.

Reply:
I guess it's ready, I'll pull this patch in https://github.com/dracutdevs/dracut/pull/871/commits/6184c7a9fed3dd75669e2545a4024ed690d9336e